docker-run

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+args=()
+for a in "$@"; do
+  escaped="$(sed "s/'/'\\\\''/g" <<< "$a")"
+  args+=("'$escaped'")
+done
+
+docker run --rm -v "$(pwd)":/genthat -w /genthat genthat:dev sh -c "${args[*]}"
```

Comment:
How about just a simple:

```sh
#!/bin/sh

docker run --rm -ti -v "$(pwd)":/genthat -w /genthat genthat:dev "$@"
```

This also allows one to enter an interactive shell. For example `./docker-run R` or `./docker-run bash` or just `./docker-run`